webonary-cloud-api/tools/flexXhtmlParser.ts

```diff
@@ -55,15 +26,20 @@ export class FlexXhtmlParser {
       this.toBeParsed.replace(/\r?\n|\r/g, '').match(/<div class="entry" (.+?)<\/div>/gm) ?? [];
 
     this.parsedItems = matches.map(
-      (entryData): Entry => {
+      (entryData): LoadEntry => {
         return FlexXhtmlParser.parseEntry(this.options.dictionary, entryData);
       },
     );
   }
 
-  static parseEntry(dictionary: string, entryData: string): Entry {
+  static parseEntry(dictionary: string, entryData: string): LoadEntry {
     const $ = cheerio.load(entryData);
-    const _id = $('div.entry').attr('id') ?? '';
+
+    // NOTE: guid field in Webonary and FLex actually includes the character 'g' at the beginning
```

Comment:
fyi this is because we used it as an 'id' in xhtml and they must always start with a letter.

Reply:
ah! makes sense. i've kept the same in wordpress, but i wanted to store in its native format for Mongo.